app/scripts/views/sign_in.js

```diff
@@ -161,7 +165,7 @@ function (Cocktail, p, BaseView, FormView, SignInTemplate, Session,
       return self.broker.afterSignIn(account)
         .then(function (result) {
           if (! (result && result.halt)) {
-            self.navigate('settings');
+            self.navigate(self._redirectTo || 'settings');
```

Comment:
I think this is accurate, but just to be explicit: the way all this backbone navigation stuff works means there's no chance of an open-redirect vulnerability here, right?